docs/sources/auth/generic-oauth.md

```diff
@@ -29,6 +29,7 @@ enabled = true
 client_id = YOUR_APP_CLIENT_ID
 client_secret = YOUR_APP_CLIENT_SECRET
 scopes =
+empty_scopes =
```

Comment:
Shouldn't this be either `false` or `true`? `empty_scopes` is supposed to be a boolean, not a (potentially empty) string.